src/app/work-item/work-item-list/work-item-list.component.scss

```diff
@@ -87,6 +87,9 @@
       border-width: 1px 0;
       padding: 1px 10px;
     }
+    .move-dropdown{
+      margin-right: em(1600);
```

Comment:
dropdown is broken. 
<img width="155" alt="screen shot 2017-01-06 at 3 33 52 pm" src="https://cloud.githubusercontent.com/assets/10396359/21714776/c6fb94d4-d426-11e6-8243-1177f4840243.png">